.gitignore

```diff
@@ -11,7 +11,18 @@
 !/.markdownlint.yml
 !/.markdownlintignore
 !/.pre-commit-config.yaml
+!/.github/PULL_REQUEST_TEMPLATE.md
+!/.github/sbom_metadata/metadata.cyclonedx.json
+!/.github/sbom_metadata/metadata.spdx.json
+!/.github/scripts/*.sh
+!/.github/workflows/*.yml
+!/.vscode/extensions.json
+!/.vscode/header.code-snippets
+!/.vscode/launch.json
+!/.vscode/settings.json
+!/.vscode/tasks.json
 !/docs/CHANGELOG.md
+!/internal/pkg/db/testdata/sbom.*.json
```

Comment:
The `testdata` folder is common in go project subdirectories and automatically ignored by `go build`. So I think this pattern might be cleaner as just `!testdata/` under the "patterns found anywhere in the project" section

It won't let me use suggestions on code that wasn't changed at the bottom of the file